packages/gatsby-plugin-graphql-config/package.json

```
@@ -0,0 +1,39 @@
+{
+  "name": "gatsby-plugin-graphql-config",
+  "description": "Gatsby plugin to write out a graphql-config with develop process endpoint configured",
+  "version": "3.2.27",
```

```suggestion
  "version": "1.0.0",
```
I think we should start with 1.0.0, no need for 0.x versions